sdk/core/azure-core/CLIENT_LIBRARY_DEVELOPER.md

```diff
@@ -193,7 +193,7 @@ The HttpRequest has the following API. It does not vary between transports:
 ```python
 class HttpRequest(object):
 
-    def __init__(self, method, url, headers=None, files=None, data=None):
+    def __init__(self, method, url, headers=None, files=None, data=None, **kwargs):
```

Comment:
Do we want to accept arbitrary arguments here?

Reply:
we're adding kwargs to the `HttpRequest` so I thought I would also add it to client library developer